mapbox_gl_platform_interface/lib/src/mapbox_gl_platform_interface.dart

```diff
@@ -19,75 +21,61 @@ abstract class MapboxGlPlatform {
   }
 
   static MapboxGlPlatform getInstance(int id) {
-    return _instances[id];
+    return _instances[id]!;
```

Comment:
Should this be nullable? I haven't dug into the logic, perhaps we can have reasonable confidence that the given id will always correspond to a value.

Reply:
`getInstance` cannot be called by the user, therefore if it is called properly internally, it should never be `null`.